pro/src/pages/Home/Homepage.tsx

```diff
@@ -185,6 +187,23 @@ export const Homepage = (): JSX.Element => {
   )
 }
 
-// Lazy-loaded by react-router-dom
+// Below exports are used by react-router-dom
 // ts-unused-exports:disable-next-line
 export const Component = Homepage
+
+export type HomepageLoaderData = { venueTypes: VenueTypeListResponseModel }
+
+// ts-unused-exports:disable-next-line
+export const loader: RouteObject['loader'] =
+  async (): Promise<HomepageLoaderData> => {
+    const venueTypes = await api.getVenueTypes()
+
+    return {
+      venueTypes,
+    }
+  }
+
+// ts-unused-exports:disable-next-line
+export const shouldRevalidate: RouteObject['shouldRevalidate'] = () => {
```

Comment:
[Cf la doc de react-router](https://reactrouter.com/en/main/route/should-revalidate), par défaut le loader est rechargé dès que l'état du router change (par exemple on change la `?structure=xxxx` dans l'url), shouldRevalidate permet d'avoir un controle plus fin sur quand on recharge ou non